service/src/main/java/uk/gov/hmcts/reform/fpl/service/FeatureToggleService.java

```diff
@@ -50,6 +50,10 @@ public boolean isExpertUIEnabled() {
         return ldClient.boolVariation("expert-ui", createLDUser(), false);
     }
 
+    public boolean isCaseUserAssignmentEnabled() {
```

Comment:
maybe it should be isCaseUserBulkAssignmentEnabled

Reply:
changed